skins/item/color/yellow.ess

```diff
@@ -0,0 +1,76 @@
+background = __.color.yellow.light
+color hsla(49.6, 73.1%, 26.3%, 1)
```

Comment:
This color is not in our palette. We'll update this later when we have more time.